src/nbl/ext/ImGui/ImGui.cpp

```diff
@@ -593,17 +596,15 @@ namespace nbl::ext::imgui
 			IMGUI_CHECKVERSION();
 			ImGui::CreateContext();
 
-			createPipeline(core::smart_refctd_ptr(_descriptorSetLayout), renderpass, pipelineCache);
+			createPipeline(core::smart_refctd_ptr(_descriptorSetLayout), renderpass, subpassIx, pipelineCache);
 			createFontAtlasTexture(transistentCMD.get(), tQueue);
 			adjustGlobalFontScale();
 		}
 		tQueue->endCapture();
 
-		static constexpr auto DEFAULT_MDI_SIZE = 1024u * 1024u * 2u; // 2 Mb
-		createMDIBuffer(DEFAULT_MDI_SIZE);
+		createMDIBuffer(mdiTotalByteSize);
```

Comment:
mdi buffer should be taken from outside and only created with default size if the outside pointer is `nullptr`

Reply:
resolved